recipe/meta.yaml

```diff
@@ -1,24 +1,21 @@
-{% set version = "16.0.0" %}
+{% set version = "16.6.2" %}
 
 package:
   name: virtualenv
   version: {{ version }}
 
 source:
   url: https://github.com/pypa/virtualenv/archive/{{ version }}.tar.gz
-  sha256: 5a597f845cf785d46c3ae39d6cb18ef0b6af4ca366da40682364f78f7fe00d22
-  patches:
-    - 0001-Mach-O-Copy-any-dynamic-libraries-and-symlinks-to-th.patch
-    - 0002-ELF-Copy-any-dynamic-libraries-and-symlinks-to-the-c.patch
```

Comment:
@ocefpaf Not sure what was the reason for not applying the 2 patches any more, but that's what is breaking the tests on Python 2.7 . In fact, for all `>=16.6.2` py27 packages, `virtualenv /tmp/venv` crashes for me.
I'll try to open a PR to restore them.

Reply:
Can you fix the credit on these patches? I wrote them.

Reply:
> Can you fix the credit on these patches? I wrote them.

Done in #23.